src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  position: relative;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: flex-end;
```

Comment:
The header should have its content justified to space-between to fit the given mockup.